src/main/java/org/jenkinsci/plugins/ghprb/GhprbRepository.java

```diff
@@ -389,7 +389,8 @@ void onPullRequestHook(PullRequest pr) throws IOException {
             doSave = true;
         } else if (!trigger.isActive()) {
             LOGGER.log(Level.FINE, "Not processing Pull request since the build is disabled");
-        } else if ("edited".equals(action) || "opened".equals(action) || "reopened".equals(action) || "synchronize".equals(action)) {
+        } else if ("edited".equals(action) || "labeled".equals(action) || "opened".equals(action)
```

Comment:
Could you move the list of allowed labels to a static defined lists and than check if the label is inside the list?

This would improve the readability :) 

Also, do we need to alter any kind of tests?